programs/programs/cross_white_label/medicaid/wa.py

```
@@ -0,0 +1,105 @@
+
+    # HCA 2026 Apple Health for Kids premium tier ceiling (317% effective FPL, annual).
+    # Source: WAC 182-505-0215; derived from 2026 HHS FPL guidelines × 3.17.
+    _PREMIUM_CHIP_ANNUAL_CEILING = {
+        1: 50_591,
+        2: 68_591,
+        3: 86_604,
+        4: 104_610,
+        5: 122_636,
+        6: 140_621,
+        7: 158_627,
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)